src/constants.js

```diff
@@ -137,6 +137,12 @@ export const INDEXED_PREPRINT_PROPS = [
 
 // See https://github.com/PREreview/rapid-prereview/issues/10
 export const DISEASES = [
+  {
+    name: '2019 novel coronavirus',
```

Comment:
@majohansson Should we have the name '2019 novel coronavirus' at all?

Reply:
This won't display in the interface, but I'm wondering if it should just be 'COVID-19' fullstop.